bindings/jni/src/main/java/org/zeromq/czmq/Zconfig.java

```diff
@@ -229,10 +229,9 @@ public void removeSubtree () {
     }
     /*
     Destroy node and subtree (all children)
-    WARNING: manually fixed void -> long
```

Comment:
please don't overwrite this - it will break the JNI bindings

Reply:
reverted the change